e2e/playwright/tests/folder-rules/page-objects/components/actions.component.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2005 - 2023 Alfresco Software, Ltd. All rights reserved.
+ *
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
+ */
+
+import { Locator, Page } from '@playwright/test';
+import { BaseComponent } from '@alfresco/playwright-shared';
+
+export enum ActionType {
+  AddAspect = 'Add aspect',
+  SimpleWorkflow = 'Add simple workflow',
+  IncrementCounter = 'Increment Counter',
+  CheckIn = 'Check in',
+  AutoDeclareOptions = 'Auto-Declare Options',
+  CheckOut = 'Check out',
+  Copy = 'Copy',
+  ExecuteScript = 'Execute script',
+  ExtractCommonMetadataFields = 'Extract common metadata fields',
+  FileAsRecord = 'File as Record',
+  FileVersionAsRecord = 'File Version as Record',
+  HideRecord = 'Hide Record',
+  Import = 'Import',
+  Move = 'Move',
+  RemoveAspect = 'Remove aspect',
+  RequestAIRenditions = 'Request AI renditions',
+  SendEmail = 'Send email',
+  SetPropertyValue = 'Set property value',
+  SpecialiseType = 'Specialise type',
+  TransformAndCopyContent = 'Transform and copy content',
+  TransformAndCopyImage = 'Transform and copy image'
+}
+
+export class ActionsDropdownComponent extends BaseComponent {
```

Comment:
Don't rename the file please, I think actions-dropdown component is more descriptive.

Reply:
Yes sure, I will revert it.